adhoc-controllers/main.go

```diff
@@ -63,7 +64,13 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
```

Comment:
I see `ctrl.GetConfigOrDie` does fallback to in-cluster if kubeconfig supplied, I think we can stick to this function

explanation about this function:
https://github.com/kubernetes-sigs/controller-runtime/blob/e42a8c17945360813e64bde2c4ff742515b750ba/alias.go#L73:L78

Reply:
Fixed.